pylabrobot/liquid_handling/liquid_handler.py

```diff
@@ -1383,6 +1399,21 @@ def save(self, path: str):
     with open(path, "w", encoding="utf-8") as f:
       json.dump(self.serialize(), f, indent=2)
 
+  def register_callback(self, method_name: str, callback: OperationCallback):
+    """Registers a callback for a specific method."""
+    if self._callbacks.get(method_name):
+      error_message = f"Callback already registered for: {method_name}"
+      raise RuntimeError(error_message)
+    self._callbacks[method_name] = callback
+
+  def _trigger_callback(self, method_name: str, *args, **kwargs):
+    """Triggers the callback associated with a method, if any."""
```

Comment:
Could you document that if a callback method exists, the error will be passed there instead of being raised?

Reply:
Added a note in the docstring, is that what you meant? Or would you like me to update the documentation?